### PR TITLE
AET client - shell script

### PR DESCRIPTION
**Pull request by michalwronowski**

### CODE REVIEW

**Comment by wiiitek** [comment]
The build has failed because the license headers are missing...
Please compare i.e. with [com.cognifide.aet.vs.mongodb.MongoDBClient.cfg](https://github.com/Cognifide/aet/blob/master/osgi-dependencies/configs/src/main/resources/com.cognifide.aet.vs.mongodb.MongoDBClient.cfg)
and add the license headers.

The license headers are checked by MAven during `validate` phase.

**Comment by michalwronowski** [comment]
Wiki page added in wiki repo in `feature/aet-client-sh` branch.

**Comment by Skejven** [comment]
@michalwronowski thank you and sorry for not having tutorial for creating AET wiki docs yet.
We keep all AET wiki docs here: 
https://github.com/Cognifide/aet/tree/master/documentation/src/documentation/src/main/wiki
Thanks to that you may add new doc files together with this pull request. After the PR is merged, `documentation/src/main/wiki` directory is merged to AET Wiki `master` branch and is available to users.
Please remember to add any new wiki page to `_Sidebar.md` - thanks to that page will appear in the Wiki ToC on the right.
If you have any assets, please simply put them to [`assets`](https://github.com/Cognifide/aet/tree/master/documentation/src/main/wiki/assets) directory.
Thanks!

File: documentation/src/main/wiki/ClientScripts.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+During test execution script creates two files:
+* `xUnit.xml` - this file is downloaded after test is processed, to extract information on any failures.
+* `redirect.html` - this file is created to help reach the test result by redirecting to test results URL after opening it.
+
+#### Dependencies
+
+In order to run properly, following commands need to be available on PATH of the environment the script is executed on:
+* `curl` - used to make actual request against [[Test Executor API|TestExecutor]] and also to download `xUnit.xml` file after test completion.
+* `jq` - used to parse JSON responses from [[Test Executor API|TestExecutor]].
```
Comment (Skejven) [inline documentation/src/main/wiki/ClientScripts.md]: Can you please include information how obtain `jq` and `xmllint` (e.g. url to download page) and write here which version of it was used?

File: documentation/src/main/wiki/ClientScripts.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+After test is completed the script downloads xUnit report to extract the test failures number. The number of failures is then set as an exit status.
+
+#### Artifacts
+
+During test execution script creates two files:
+* `xUnit.xml` - this file is downloaded after test is processed, to extract information on any failures.
+* `redirect.html` - this file is created to help reach the test result by redirecting to test results URL after opening it.
+
+#### Dependencies
```
Comment (Skejven) [inline documentation/src/main/wiki/ClientScripts.md]: Or we can name it `Prerequisites`

**Pull request merged**